scripts/webpack/webpack-resources.js

```diff
@@ -79,7 +79,7 @@ module.exports = {
         devServer: {
           inline: true,
           port: 4322,
-          contentBase: path.resolve(process.cwd(), 'dist')
+          contentBase: path.resolve(process.cwd())
```

Comment:
Have you verified that this doesn't break serve in any of the other apps?

Reply:
I checked in several apps and it wasn't breaking them, but I'll try each one and get to you with the results afterwards.

Reply:
Thanks! You should probably also talk to David and/or Ken to see if they know why the path was this way to begin with and what might break if it's changed.

Reply:
I reverted this change and fixed the root of the issue, which was that a `pre-copy.json` needed to be added to the project's `config` folder.